src/StarterKitServiceProvider.php

```diff
@@ -13,7 +13,7 @@ class StarterKitServiceProvider extends PackageServiceProvider
 {
     const PACKAGE_NAME = 'starterkit';
 
-    const THEME_NAME = 'cwd-framework';
+    const THEME_NAME = 'cd';
```

Comment:
Per my comment above, we could perhaps add a constant for the components:
```
const COMPONENT_DIR = 'cd';
```
and then use that for the files that are published to the resources directory. Then restore the THEME_NAME value to the theme we are supplying: 'cwd-framework'.

Reply:
What is the theme name used for other than the directory name?  Maybe we don't need THEME_NAME if we use COMPONENT_DIR.